protocol-spec/enumeration.md

```diff
@@ -4,6 +4,8 @@
 
 **Description:** Client request to have the server start scanning for devices on all busses that it knows about. Useful for protocols like Bluetooth, which require an explicit discovery phase.
 
+**Introduced In Version:** 0
```

Comment:
Actually 0.1.0 was message version 0 (because that's what the 0.1.x server announced it was working with in the serverinfo response).

The relationship between release numbers and message versions is in the messages.md#message-versioning section

Reply:
Ok, let's leave it 0 and 1, but remove the version mapping and just call it by the spec version from here on out. These should not have to match the project versions. There will many implementations with many versioning schemes.

Reply:
Cool. Should I update the schema to match?